doc/user-changes.md

```diff
@@ -6,6 +6,19 @@ stay on top of `alr` new features.
 
 ## Release `2.0-dev`
 
+### Deprecation of `toolchain --install/--uninstall/--install-dir`
+
+PR [#1614](https://github.com/alire-project/alire/pull/1614)
+
+Toolchain installation for use by Alire is still possible using
+`alr toolchain --select`. For the installation of toolchains outside of Alire
+management, `alr get` covers the `alr toolchain --install --install-dir` use
```

Comment:
Same as below, why not recommend `alr install --prefix` here?

Reply:
Again, because for people that was using `--install-dir`, this is what they would find more similar. But if we want people to just use `alr install`, we can omit `alr get` altogether.